react/week4/src/components/Headers.js

```diff
@@ -0,0 +1,17 @@
+import './Headers.css'
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fab } from '@fortawesome/free-brands-svg-icons';
+
+library.add(fab)
+
+export default function MainTitle() {
+    return <h1><FontAwesomeIcon icon={['fab', 'github']} /> GitHub Users</h1>
+}
+
+function SectionTitle(props) {
+    return <h3 className={props.className}>{props.title}</h3>
+}
```

Comment:
I would say that you don't need those components. You only have two titles in general in the whole app that are also not changeable. It would also ok if you add h3 as part of UsersList component, and h1 as part of UsersSearch component.